src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -278,6 +282,7 @@ const showToast = (title, message, variant, mode, messageData) => {
 * @param {object} error: Event holding error details
 */
 const handleError = (error) => {
+    console.log('error', error);
```

Comment:
We generally should be pulling out console logs before merging to the feature branch